app/src/lib/git/core.ts

```diff
@@ -101,6 +101,10 @@ export async function git(args: string[], path: string, name: string, options?:
   const defaultOptions: IGitExecutionOptions = {
     successExitCodes: new Set([ 0 ]),
     expectedErrors: new Set(),
+    env: {
+      GIT_TERMINAL_PROMPT: '0',
```

Comment:
We're already passing this in for anything authentication-related here: https://github.com/desktop/desktop/blob/master/app/src/lib/git/core.ts#L227-L234

Do we just need to do this for committing? Could we conditionalize it there? Or do you think this is worth setting everywhere?